pytest.ini

```
@@ -0,0 +1,4 @@
+[pytest]
+addopts = --ignore=pic_sleuth/config/*
+DJANGO_SETTINGS_MODULE = pic_sleuth.config.settings
+norecursedirs = pic_sleuth/*/migrations/*
```

We've ignored using `coverage:run` instead.

While we're here, you can tell it not to look at the env folders, etc.

```suggestion
norecursedirs = venv build .env bin .cache .tox
```

---

done